steps/dexprotector/1.0.1/step.yml

```diff
@@ -0,0 +1,191 @@
+title: DexProtector
+summary: |
+  Protect your Android or iOS app with DexProtector.
+description: |
+  Protect your Android or iOS app with DexProtector.
+  This Step allows you to integrate DexProtector into your Android or iOS app build as simply as possible using Bitrise.
+  Note: To use this Step, you need a valid DexProtector Enterprise Site license.
+  For more information, reach out to us at primary@licelus.com or visit [licelus.com](https://licelus.com/).
+
+  First, make sure that Bitrise has access to a repository containing all of the following:
+  your project’s source code; DexProtector (dexprotector.jar);
+  your DexProtector configuration file (dexprotector.xml).
+
+  You will also need the Base64 string of your DexProtector license file.
+  This string must be stored in the Secrets tab of the Bitrise Workflow Editor,
+  and passed from there as an input to this Step.
+
+  Visit [licelus.com](https://licelus.com/) for more information.
+  For licensing or other queries, please contact us at primary@licelus.com.
+
+  Example of project structure:
+  ```
+  /
+  +-- dexprotector
+  |   +-- dexprotector.jar
+  |   +-- dexprotector.xml
+  [YOUR PROJECT FILES]
+  ```
+website: https://github.com/dexprotector/bitrise-step-dexprotector
+source_code_url: https://github.com/dexprotector/bitrise-step-dexprotector
+support_url: https://github.com/dexprotector/bitrise-step-dexprotector/issues
+published_at: 2021-04-26T18:42:25.769554+08:00
+source:
+  git: https://github.com/dexprotector/bitrise-step-dexprotector.git
+  commit: 73ee52a2f25791aa6fe43e85f392c2ee08402943
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- ios
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: java
+  apt_get:
+  - name: java
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- app_file: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: 'Path to the app file to be protected. For example: `$BITRISE_APK_PATH`
+      for Android; `$BITRISE_IPA_PATH` for iOS.'
```

Comment:
What about AAB? Is it supported?

Reply:
Yes, we are support APK, AAB and AAR Android applications.`$BITRISE_APK_PATH` here just for example.

Reply:
**[SUGGESTION]** you could potentially mention this in the documentation somewhere :) maybe in the description or the parameter summary, just to make it clear for the consumer that these are supported.